builtInFunctions/esdtMetaDataUpdate.go

```diff
@@ -150,6 +157,16 @@ func (e *esdtMetaDataUpdate) SetNewGasConfig(gasCost *vmcommon.GasCost) {
 	e.mutExecution.Unlock()
 }
 
+// SetBlockDataHandler is called when block data handler is set
+func (e *esdtMetaDataUpdate) SetBlockDataHandler(blockDataHandler vmcommon.BlockDataHandler) error {
```

Comment:
unit tests for all SetBlockDataHandler ?

Reply:
Added.